UnitTests/GitUI.Tests/ControlThreadingExtensionsTests.cs

```diff
@@ -171,6 +171,7 @@ public async Task TokenCancelledBeforeSwitchOnBackgroundThread()
         }
 
         [Test]
+        [Ignore("Hangs")]
         public async Task ControlDisposedAfterSwitchOnBackgroundThread()
```

Comment:
@sharwell this test hangs run on .NET Core 3.1 in: `await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();`.
Do you think you can have a look?